tests/manage/mcg/test_write_to_bucket.py

```diff
@@ -70,10 +70,68 @@ def test_write_file_to_bucket(self, mcg_obj, awscli_pod, bucket_factory):
     @pytest.mark.polarion_id("OCS-1949")
     @tier1
     @acceptance
-    def test_data_reduction(self, mcg_obj, awscli_pod, bucket_factory):
+    def test_data_deduplication(self, mcg_obj, awscli_pod, bucket_factory):
```

Comment:
Since you're using test-specific directories (/aws/dedup... /aws/reduction) - I think it's a great chance to use awscli_pod_session